dags/taar_amodump.py

```diff
@@ -1,83 +1,122 @@
 
 default_args = {
-    'owner': 'amiyaguchi@mozilla.com',
-    'depends_on_past': False,
-    'start_date': datetime(2018, 11, 26),
-    'email': ['telemetry-alerts@mozilla.com', 'amiyaguchi@mozilla.com'],
-    'email_on_failure': True,
-    'email_on_retry': True,
-    'retries': 3,
-    'retry_delay': timedelta(minutes=30),
+    "owner": "vng@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 10, 7),
+    "email": ["telemetry-alerts@mozilla.com", "amiyaguchi@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 3,
+    "retry_delay": timedelta(minutes=30),
 }
 
-dag = DAG('taar_amodump', default_args=default_args, schedule_interval='@daily')
+dag = DAG("taar_amodump", default_args=default_args, schedule_interval="@daily")
 
-amodump = EMRSparkOperator(
+amodump = GKEPodOperator(
     task_id="taar_amodump",
-    job_name="Dump AMO JSON blobs with oldest creation date",
-    execution_timeout=timedelta(hours=1),
-    instance_count=1,
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    name="taar-amodump",
+    namespace="default",
+    image="gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.1",
     owner="vng@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_amodump",
-                      {"date": "{{ ds_nodash }}"},
-                      {'MOZETL_SUBMISSION_METHOD': 'python'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    email=["mlopatka@mozilla.com", "vng@mozilla.com", "hwoo@mozilla.com"],
+    arguments=["-m", "taar_etl.taar_amodump", "--date", "{{ ds_nodash }}"],
+    env_vars={
+        "AWS_ACCESS_KEY_ID": aws_access_key,
+        "AWS_SECRET_ACCESS_KEY": aws_secret_key,
+    },
+    dag=dag,
 )
 
-amowhitelist = EMRSparkOperator(
+amowhitelist = GKEPodOperator(
     task_id="taar_amowhitelist",
-    job_name="Generate an algorithmically defined set of whitelisted addons for TAAR",
-    execution_timeout=timedelta(hours=1),
-    instance_count=1,
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    location="us-central1-a",
+    cluster_name="bq-load-gke-1",
+    name="taar-amowhitelist",
+    namespace="default",
+    image="gcr.io/moz-fx-data-airflow-prod-88e0/taar_gcp_etl:0.1",
     owner="vng@mozilla.com",
-    email=["mlopatka@mozilla.com", "vng@mozilla.com"],
-    env=mozetl_envvar("taar_amowhitelist",
-                      {},
-                      {'MOZETL_SUBMISSION_METHOD': 'spark'}),
-    uri="https://raw.githubusercontent.com/mozilla/python_mozetl/master/bin/mozetl-submit.sh",
-    output_visibility="private",
-    dag=dag
+    email=["mlopatka@mozilla.com", "vng@mozilla.com", "hwoo@mozilla.com"],
+    arguments=["-m", "taar_etl.taar_amowhitelist"],
```

Comment:
No date req?

Reply:
Nope. But added comments about it.  

The job just crawls the AMO API site to build a list of available addons.  Adding a date parameter wouldn't give us a real date scoped list of addons as AMO is always going to give us the latest available addon list.